example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -25,8 +24,7 @@ protected boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage(),
-          new RNSvgPackage()
```

Comment:
This automated change has caused the example app to break in the past. Did you get a chance to run through the example app successfully in an Android simulator after you made these changes? If its broken, we probably will need to manually add back in the `new RNSvgPackage()` to make it work